src/backends/postgresql/row-id.cpp

```diff
@@ -31,7 +31,7 @@ using namespace soci::details;
 postgresql_rowid_backend::postgresql_rowid_backend(
     postgresql_session_backend & /* session */)
 {
```

Comment:
Good point, initializer lists are indeed more SOCI style. I still prefer to keep an explicit 0 however for clarity.